charts/consul/Chart.yaml

```diff
@@ -10,12 +10,12 @@ sources:
   - https://github.com/hashicorp/consul
   - https://github.com/hashicorp/consul-k8s
 annotations:
-  artifacthub.io/prerelease: false
+  artifacthub.io/prerelease: true
   artifacthub.io/images: |
     - name: consul
       image: hashicorp/consul:1.13.1
```

Comment:
```suggestion
      image: hashicorp/consul:1.13.2
```

Reply:
ah yes, this isn't part of the automation. This step is done manually atm, it'll make more sense with the release process revision coming to the engineering docs soon!